<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UserInputDialog</class>
 <widget class="QDialog" name="UserInputDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>866</width>
    <height>355</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>866</width>
    <height>354</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>866</width>
    <height>355</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="TitleBar" name="widget" native="true">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>55</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>55</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">#widget
{
border-bottom:2px solid #282C36;
}</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget_2" native="true">
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QWidget" name="widget_3" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">#widget_3
{
border-bottom:2px solid #282C36;
}</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>84</number>
         </property>
         <property name="topMargin">
          <number>21</number>
         </property>
         <property name="rightMargin">
          <number>83</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <layout class="QGridLayout" name="gridLayout">
           <property name="horizontalSpacing">
            <number>80</number>
           </property>
           <property name="verticalSpacing">
            <number>18</number>
           </property>
           <item row="0" column="0">
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <property name="spacing">
              <number>0</number>
             </property>
             <item>
              <widget class="QLabel" name="label_4">
               <property name="text">
                <string>账号：</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="lineEdit_id">
               <property name="minimumSize">
                <size>
                 <width>252</width>
                 <height>33</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>252</width>
                 <height>33</height>
                </size>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item row="0" column="1">
            <layout class="QHBoxLayout" name="horizontalLayout_5">
             <property name="spacing">
              <number>0</number>
             </property>
             <item>
              <widget class="QLabel" name="label">
               <property name="text">
                <string>姓名：</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="lineEdit_name">
               <property name="minimumSize">
                <size>
                 <width>252</width>
                 <height>33</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>252</width>
                 <height>33</height>
                </size>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item row="1" column="0">
            <layout class="QHBoxLayout" name="horizontalLayout_3">
             <property name="spacing">
              <number>0</number>
             </property>
             <item>
              <widget class="QLabel" name="label_3">
               <property name="text">
                <string>密码：</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="lineEdit_password">
               <property name="minimumSize">
                <size>
                 <width>252</width>
                 <height>33</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>252</width>
                 <height>33</height>
                </size>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item row="1" column="1">
            <layout class="QHBoxLayout" name="horizontalLayout_6">
             <property name="spacing">
              <number>0</number>
             </property>
             <item>
              <widget class="QLabel" name="label_2">
               <property name="text">
                <string>性别：</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="comboBox_sex">
               <property name="minimumSize">
                <size>
                 <width>252</width>
                 <height>33</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>252</width>
                 <height>33</height>
                </size>
               </property>
               <item>
                <property name="text">
                 <string>男</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>女</string>
                </property>
               </item>
              </widget>
             </item>
            </layout>
           </item>
           <item row="2" column="0">
            <layout class="QHBoxLayout" name="horizontalLayout_4">
             <property name="spacing">
              <number>0</number>
             </property>
             <item>
              <widget class="QLabel" name="label_5">
               <property name="text">
                <string>权限：</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="comboBox_authority">
               <property name="minimumSize">
                <size>
                 <width>252</width>
                 <height>33</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>252</width>
                 <height>33</height>
                </size>
               </property>
               <item>
                <property name="text">
                 <string>普通用户</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>管理员</string>
                </property>
               </item>
              </widget>
             </item>
            </layout>
           </item>
           <item row="2" column="1">
            <layout class="QHBoxLayout" name="horizontalLayout_7">
             <property name="spacing">
              <number>0</number>
             </property>
             <item>
              <widget class="QLabel" name="label_6">
               <property name="maximumSize">
                <size>
                 <width>16777215</width>
                 <height>33</height>
                </size>
               </property>
               <property name="text">
                <string>状态：</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="comboBox_enable">
               <property name="minimumSize">
                <size>
                 <width>252</width>
                 <height>33</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>252</width>
                 <height>33</height>
                </size>
               </property>
               <item>
                <property name="text">
                 <string>可用</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>不可用</string>
                </property>
               </item>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
         <item>
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Fixed</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>56</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <property name="spacing">
         <number>24</number>
        </property>
        <property name="topMargin">
         <number>20</number>
        </property>
        <property name="rightMargin">
         <number>22</number>
        </property>
        <property name="bottomMargin">
         <number>20</number>
        </property>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_add">
          <property name="minimumSize">
           <size>
            <width>96</width>
            <height>34</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>96</width>
            <height>34</height>
           </size>
          </property>
          <property name="text">
           <string>添加</string>
          </property>
          <property name="icon">
           <iconset>
            <normaloff>:/image/confirm.png</normaloff>:/image/confirm.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>17</width>
            <height>17</height>
           </size>
          </property>
          <property name="class" stdset="0">
           <string>BlueButton</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_close">
          <property name="minimumSize">
           <size>
            <width>96</width>
            <height>34</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>96</width>
            <height>34</height>
           </size>
          </property>
          <property name="text">
           <string>关闭</string>
          </property>
          <property name="icon">
           <iconset>
            <normaloff>:/image/circle_close.png</normaloff>:/image/circle_close.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>17</width>
            <height>17</height>
           </size>
          </property>
          <property name="class" stdset="0">
           <string>BlackButton</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>TitleBar</class>
   <extends>QWidget</extends>
   <header>titlebar.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../res/qr.qrc"/>
 </resources>
 <connections/>
</ui>
